tests/RewriteTest.php

```diff
@@ -23,6 +23,7 @@ public function testOnCloudFlareIPv4()
         $this->ipRewrite = new IpRewrite();
 
         $this->assertTrue($this->ipRewrite->isCloudFlare());
+        $this->assertTrue($this->ipRewrite->isCloudFlareIP());
```

Comment:
we should have a test where `isCloudFlare` is `false` where as `isCloudFlareIp` is `true`.